cv32/sim/uvmt_cv32/Makefile

```diff
@@ -115,8 +115,8 @@ DSIM_IMAGE              = dsim.out
 XRUN              = xrun
 #XRUN_UVMHOME_ARG ?= CDNS-1.2
 XRUN_UVMHOME_ARG ?= CDNS-1.2-ML
-#XRUN_FLAGS       ?= -64bit -access +rwc -q -clean -smartorder -sv -uvm -uvmhome $(XRUN_UVMHOME_ARG) -top worklib.uvmt_cv32_tb -timescale 1ns/1ps
-XRUN_FLAGS       ?= -64bit -access +rwc -q -clean -sv -uvm -uvmhome $(XRUN_UVMHOME_ARG) -timescale 1ns/1ps
+#XRUN_FLAGS       ?= -64bit -access +rwc -q -clean -smartorder -sv -uvm -disable_sem2009 -uvmhome $(XRUN_UVMHOME_ARG) -top worklib.uvmt_cv32_tb -timescale 1ns/1ps
+XRUN_FLAGS       ?= -64bit -access +rwc -q -clean -sv -uvm -disable_sem2009 -uvmhome $(XRUN_UVMHOME_ARG) -timescale 1ns/1ps
```

Comment:
Similar comment as above.

Note that the structure of the Makefiles for the UVM environment hves been significantly changed such that simulator dependent variables/targets/rules have their own file.  So the variable XRUN_FLAGS is no longer in the Makefile, it is in `xrun.mk`, which is included by the Makefile.

Reply:
Okay